package.json

```diff
@@ -13,6 +13,7 @@
     "build.example.angular": "pnpm --filter component-library-angular run build",
     "build.example.core": "pnpm --filter component-library run build",
     "build.example.react": "pnpm --filter component-library-react run build",
+    "build.example.react-esm": "pnpm --filter component-library-react-esm run build",
```

Comment:
I think having the unit tests is sufficient for this. I want to avoid having too many integration tests as they slow down the pipeline significantly.

Reply:
updated

Reply:
Ah, you want to delete the entire package? Got it. Will do